lib/workers/repository/process/extract-update.ts

```diff
@@ -147,6 +168,7 @@ export async function extract(
       configHash,
       extractionFingerprints,
       packageFiles,
+      onboardingSha: getBranchCommit(config.onboardingBranch!),
```

Comment:
Try to set this only if the repo is onboarding

Reply:
I am thinking of merging https://github.com/renovatebot/renovate/pull/20886, first. That way we won't have to create `onboardingBranchCache`.

Reply:
Sure, if you're confident in the cache structure